Include/pyport.h

```diff
@@ -101,7 +101,9 @@ typedef intptr_t        Py_intptr_t;
  * sizeof(size_t).  C99 doesn't define such a thing directly (size_t is an
  * unsigned integral type).  See PEP 353 for details.
  */
-#ifdef HAVE_SSIZE_T
+#ifdef HAVE_PY_SSIZE_T
+
+#elif HAVE_SSIZE_T
```

Comment:
This seems different than what the patch originally suggested. Out of curiosity, why is that?
From the patchfile on the bpo:
```patch
 #ifdef HAVE_SSIZE_T
 typedef ssize_t         Py_ssize_t;
+#define HAVE_PY_SSIZE_T 1
 #elif SIZEOF_VOID_P == SIZEOF_SIZE_T
 typedef Py_intptr_t     Py_ssize_t;
-#else
+#define HAVE_PY_SSIZE_T 1
+#endif
+
+#ifndef HAVE_PY_SSIZE_T
```

Reply:
The reason is this message in the issue tracker: https://bugs.python.org/msg236207

Reply:
Ahh ok, thanks for telling me :).